cmd/bff/main.go

```diff
@@ -69,3 +89,35 @@ func serve(c *cli.Context) (err error) {
 	}
 	return nil
 }
+
+// Status checks if the GDS BFF is up
+func status(c *cli.Context) (err error) {
+	var client api.BFFClient
+	if client, err = api.New(c.String("url")); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
+	defer cancel()
+
+	var rep *api.StatusReply
+	if rep, err = client.Status(ctx); err != nil {
+		return cli.Exit(err, 1)
+	}
+
+	return printJSON(rep)
+}
```

Comment:
This is usually very helpful for devops and debugging, so I wanted to make sure it was in the CLI program.